### PR TITLE
Feature/hw 66628 v4 java sdk venmo account

### PR DESCRIPTION
**Pull request by jkurra-hw**

### CODE REVIEW

**Review by Rrathinasabapath — APPROVED** [review]
Reviewed the changes.

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 1]
```diff
@@ -1131,6 +1131,215 @@ public HyperwalletStatusTransition getPayPalAccountStatusTransition(String userT
...
+        venmoAccount.setStatus(null);
+        venmoAccount.setCreatedOn(null);
+        return apiClient.post(url + "/users/" + venmoAccount.getUserToken() + "/venmo-accounts", venmoAccount, HyperwalletVenmoAccount.class);
+    }
+
+    /**
+     * Get Venmo Account
+     *
+     * @param userToken         User token assigned
+     * @param venmoAccountToken venmo Account token assigned
```
Comment (arao6) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:1176]: Upper case venmo Account

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 2]
```diff
@@ -1131,6 +1131,215 @@ public HyperwalletStatusTransition getPayPalAccountStatusTransition(String userT
...
+    }
+
+    /**
+     * Create Venmo Account Status Transition
+     *
+     * @param userToken         User token
+     * @param venmoAccountToken Venmo account token
+     * @param transition        Status transition information
+     * @return HyperwalletStatusTransition new status for venmo Account
+     */
```
Comment (arao6) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:1267]: Uppercase venmo

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 3]
```diff
@@ -1131,6 +1131,215 @@ public HyperwalletStatusTransition getPayPalAccountStatusTransition(String userT
...
+                HyperwalletVenmoAccount.class);
+    }
+
+    /**
+     * Deactivate Venmo Account
+     *
+     * @param userToken         User token
+     * @param venmoAccountToken Venmo account token
+     * @return HyperwalletStatusTransition deactivated venmo account
+     */
```
Comment (arao6) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:1241]: Uppercase venmo

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java (hunk 1 above)
Comment (jkurra-hw) [reply src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java]: Addressed it . @arao6

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java (hunk 3 above)
Comment (jkurra-hw) [reply src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:1309]: done.

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java (hunk 2 above)
Comment (jkurra-hw) [reply src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:1335]: changed to Uppercase

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 4]
```diff
@@ -1997,4 +2206,9 @@ private HyperwalletPayPalAccount copy(HyperwalletPayPalAccount payPalAccount) {
         return payPalAccount;
     }
 
+    private HyperwalletVenmoAccount copy(HyperwalletVenmoAccount venmoAccount) {
+        venmoAccount = HyperwalletJsonUtil.fromJson(HyperwalletJsonUtil.toJson(venmoAccount), HyperwalletVenmoAccount.class);
```
Comment (peter-joseph) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:2210]: we can directly return from here if we are not going to use `venmoAccount` variable

**Pull request merged**